packages/extension-ui/package.json

```diff
@@ -36,6 +36,7 @@
     "@polkadot/ui-settings": "^2.9.15",
     "@polkadot/util": "^12.1.2",
     "@polkadot/util-crypto": "^12.1.2",
+    "bignumber.js": "^9.1.1",
```

Comment:
Not using any kind of _big number_ library before was a big mistake - numbers would start turning silently incorrect at around 9k units (9k * 10^12 ~= `Number.MAX_SAFE_INTEGER`).